build.gradle

```diff
@@ -1,38 +1,81 @@
 plugins {
-  id "com.jfrog.bintray" version "1.8.4"
+    id "com.jfrog.bintray" version "1.8.4"
```

Comment:
I don't think we will be needing this anymore

Reply:
Good spot, have removed